api/apps/api/src/modules/scenarios/blm-calibration/create-initial-blm.command.ts

```diff
@@ -0,0 +1,17 @@
+import { Command } from '@nestjs-architects/typed-cqrs';
+import { Either } from 'fp-ts/Either';
+
+export const blmCreationFailure = Symbol('could not create BLM values');
+export type CreationFailure = typeof blmCreationFailure;
+export type CreationSuccess = true;
+
+export class CreateInitialBlm extends Command<
```

Comment:
Should we rename this command to `CreateInitialScenarioBlm`? It's more specific right?